README.md

```diff
@@ -34,13 +34,24 @@ When the connector switches to CDC mode, it starts streaming changes from the
-| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- | --------------------------------------------------------------- |
-| `dsn`                                    | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database.                          | true     |               | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true       |
-| `tables`                                 | The list of tables to pull data from                                                                                                                    | true     |               | users,posts,admins                                              |
-| `tableConfig.<table name>.sortingColumn` | The custom column to use to sort the rows during the snapshot. Use this if there are any tables which don't have a proper autoincrementing primary key. |          |               | `tableConfig.users.sortingColumn` as the key, `id` as the value |
+| name                                        | description                                                                                                                                             | required | default | example                                                         |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- | --------------------------------------------------------------- |
+| `dsn`                                       | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database.                          | true     |         | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true       |
+| `tables`                                    | The list of tables to pull data from                                                                                                                    | true     |         | users,posts,admins                                              |
+| `tableConfig.<table name>.sortingColum (*)` | The custom column to use to sort the rows during the snapshot. Use this if there are any tables which don't have a proper autoincrementing primary key. | false    |         | `tableConfig.users.sortingColumn` as the key, `id` as the value |
+| `fetchSize`                                 | Limits how many rows should be retrieved on each database fetch.                                                                                        | false    | 50000   | 50000                                                           |
```

Comment:
At least to me, the default fetch size looks a bit large. How did we establish it?

Reply:
I took it from the [postgres connector](https://github.com/ConduitIO/conduit-connector-postgres/blob/9cb960ef561d097f57930e27dce8335fff11ab2c/source/config.go#L63). 

Claude tells me that's too big, so I don't know, should we change it?

![image](https://github.com/user-attachments/assets/4d986a53-db78-44ca-9080-aa4e9d994494)

Reply:
I guess there must have been a reason to have such a high default, but IMO, it would be safer to have a smaller default. In Debezium, the snapshot batch size is 10k. Personally, I wouldn't go over that.